test/UnitTest/Microsoft.Test.AspNet.OData/Formatter/Serialization/ODataResourceSetSerializerTests.cs

```diff
@@ -730,8 +749,38 @@ private static void AssertEqual(Uri expected, Uri actual)
             Assert.Equal(expected.AbsoluteUri, actual.AbsoluteUri);
         }
 
-#if NETFX // Only AspNet version has UrlHelper
-        private static UrlHelper CreateMetadataLinkFactory(string metadataPath)
+#if NETCORE
+        public class MyUrlHelper : IUrlHelper
+        {
+            private string _link;
```

Comment:
Stylecop/Code Analysis may complain about starting with underscore in variable names.